spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerBusAutoConfiguration.java

```diff
@@ -0,0 +1,142 @@
+                .configuration(axonServerConfiguration)
+                .localSegment(localSegment)
+                .serializer(messageSerializer)
+                .routingStrategy(routingStrategy)
+                .priorityCalculator(priorityCalculator)
+                .loadFactorProvider(loadFactorProvider)
+                .targetContextResolver(targetContextResolver)
+                .build();
+    }
+
+    @Bean
+    @ConditionalOnMissingBean(QueryBus.class)
+    public AxonServerQueryBus queryBus(AxonServerConnectionManager axonServerConnectionManager,
+                                       AxonServerConfiguration axonServerConfiguration,
+                                       AxonConfiguration axonConfiguration,
+                                       TransactionManager txManager,
+                                       @Qualifier("messageSerializer") Serializer messageSerializer,
+                                       Serializer genericSerializer,
+                                       QueryPriorityCalculator priorityCalculator,
+                                       QueryInvocationErrorHandler queryInvocationErrorHandler,
+                                       TargetContextResolver<? super QueryMessage<?, ?>> targetContextResolver) {
+        SimpleQueryBus simpleQueryBus =
+                SimpleQueryBus.builder()
+                        .messageMonitor(axonConfiguration.messageMonitor(QueryBus.class, "queryBus"))
+                        .transactionManager(txManager)
+                        .queryUpdateEmitter(axonConfiguration.getComponent(QueryUpdateEmitter.class))
+                        .errorHandler(queryInvocationErrorHandler)
+                        .build();
+        simpleQueryBus.registerHandlerInterceptor(
+                new CorrelationDataInterceptor<>(axonConfiguration.correlationDataProviders())
+        );
+
+        return AxonServerQueryBus.builder()
+                .axonServerConnectionManager(axonServerConnectionManager)
+                .configuration(axonServerConfiguration)
+                .localSegment(simpleQueryBus)
+                .updateEmitter(simpleQueryBus.queryUpdateEmitter())
+                .messageSerializer(messageSerializer)
+                .genericSerializer(genericSerializer)
+                .priorityCalculator(priorityCalculator)
+                .targetContextResolver(targetContextResolver)
+                .build();
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public EventStore eventStore(AxonServerConfiguration axonServerConfiguration,
+                                 AxonConfiguration configuration,
+                                 AxonServerConnectionManager axonServerConnectionManager,
+                                 Serializer snapshotSerializer,
+                                 @Qualifier("eventSerializer") Serializer eventSerializer) {
+        return AxonServerEventStore.builder()
```

Comment:
Nit: I'd expect `.messageMonitor(configuration.messageMonitor(AxonServerEventStore.class, "eventStore"))` to ident to `AxonServerEventStore.`.